Fix blank blaze campaign detail page

irfano opened the pull request:
<!-- Remember about a good descriptive title. -->

<!-- Id number of the Linear issue this PR addresses, e.g., WOOMOB-373. -->

### Description
<!-- Take the time to write a good summary. Why is it needed? What does it do? When fixing bugs try to avoid just writing “See original issue” – clarify what the problem was and how you’ve fixed it. -->
This commit (84f20ad990162f614d4b2d154f4c928253391742) in 23.6 introduced an issue where the Blaze campaign detail page appeared blank. The commit added multi-window (popup) support, but this caused the WebView to open a blank popup for the Blaze campaign page.
This PR fixes the issue by disabling popups specifically for Blaze screens.
This new parameter `disablePopups` also fixes WebViews in CIAB admin and product pages and I'll fix it for CIAB sites next week. (p1763397615377999-slack-C03L1NF1EA3)

### Test Steps
<!-- Describe how to test your changes. Include only what’s needed for the reviewer to validate the behavior.
For new features, outline the main user flow or goal rather than every tap or click — the reviewer should be able to complete the flow naturally.
If applicable, mention key devices, scenarios, or edge cases to verify. -->
1. Test "Test steps in #14827

> 1. Use a site:
>     - Supports Jetpack SSO (either atomic site, or self-hosted site with Jetpack SSO enabled from the settings)
>     - WooPayments setup is not done yet, or use the Reset account button.
> 3. Open the Payments hub in the app.
> 4. Start the setup from the app.
> 5. Confirm it uses the Authenticated WebView, and the Stripe onboarding screen is opened without issues.

2. Open the detail screen of a Blaze campaign.

### Images/gif
<!-- Include before and after images or gifs when appropriate. -->
|Before|After|
|-|-|
|<img width="300" src="https://github.com/user-attachments/assets/a6058cfd-b139-42a6-acdb-772363f12b91" />|<img width="300" src="https://github.com/user-attachments/assets/9148ca76-0cce-4941-b865-7675690a5fa3" />|

- [x] I have considered if this change warrants release notes and have added them to `RELEASE-NOTES.txt` if necessary. Use the "[Internal]" label for non-user-facing changes.

<!-- Pull request guidelines: https://github.com/woocommerce/woocommerce-android/blob/develop/docs/pull-request-guidelines.md -->

irfano committed:
Disable popups in Blaze campaign detail screen

dangermattic commented:
<!--
  0 Errors
  1 Warning: This PR is assigned to the mil...
  1 Message: This PR contains changes to <c...
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/woocommerce/woocommerce-android/milestone/275">23.7 ❄️</a>. The due date for this milestone has already passed.<br />
Please assign it to a milestone with a later deadline or check whether the release for this milestone has already been finished.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR contains changes to <code>RELEASE-NOTES.txt</code>.<br />
Note that these changes won't affect the final version of the release notes as this version is in code freeze.<br />
Please, get in touch with a release manager if you want to update the final release notes.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

irfano committed:
Update RELEASE-NOTES.txt

JorgeMucientes reviewed (approved):
Great job surfacing this bugs and finding the root cause @irfano 👏🏼 . Also nice that you manage to find a solution that doesn't involve reverting all the changes. 

On thing that it's unclear to me is this: 
> This also fixes WebViews in CIAB admin and product pages. (p1763397615377999-slack-C03L1NF1EA3)

Is that so? Do you mean in this PR? Because I don't see any changes here to disable popups when opening product detail in a webview for bookable products. And I tested it and it still doesn't work.

irfano commented:
> Is that so? Do you mean in this PR? Because I don't see any changes here to disable popups when opening product detail in a webview for bookable products. And I tested it and it still doesn't work.

Oh, sorry, you’re right. My first attempt was disabling popups for all screens, and that’s how I noticed it was also fixed for CIAB sites. But this PR only affects the Blaze detail screen.
I’ll open a PR to use the `disablePopups` parameter for Booking product and admin pages next week.

I updated my comment in PR description.
